http3/response_writer.go

```diff
@@ -220,8 +246,19 @@ func (w *responseWriter) FlushError() error {
 	if !w.headerComplete {
 		w.WriteHeader(http.StatusOK)
 	}
-	_, err := w.doWrite(nil)
-	return err
+	if _, err := w.doWrite(nil); err != nil {
+		return err
+	}
+	return nil
+}
+
+func (w *responseWriter) flushFinal() {
```

Comment:
I added this function because developers should be able to call Flush() several times in a handler, so we can't actually send the trailers after each flush. This function is just intended to be called at the very end of the request, called by `http3.Server`.

Reply:
Is this because it's valid to call `http.ResponseWriter.Flush` before sending the entire response body?

Maybe make the name more descriptive?
```suggestion
func (w *responseWriter) flushTrailers() {
```